clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -207,7 +230,7 @@ public void testResetPositionsProcessFailureIsIgnored() {
 
         ResetPositionsEvent event = new ResetPositionsEvent(calculateDeadlineMs(time, 100));
         applicationEventsQueue.add(event);
-        assertDoesNotThrow(() -> consumerNetworkThread.runOnce());
+        assertDoesNotThrow(consumerNetworkThread::runOnce);
```

Comment:
can we revert the irrelevant change? also I think we can remove this test and see if this is covered elsewhere.